th/2012/spring.json

```diff
@@ -0,0 +1,777 @@
+			"detail": [{
+				"name": "Elaborate Production"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Genshokyou-fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Go-Baka FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "GumEiEiBye-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "NoGod-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Penguin-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Suzukaze-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "LuckySubs"
+			}, {
+				"name": "YRanger"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "idoli-z"
+			}]
+		}]
+	}, {
+		"name": "Jormungand",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/11/73280.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "APP"
+			}, {
+				"name": "DSG"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Kirina-Fs"
+			}, {
+				"name": "The Anime-FS"
+			}]
+		}]
+	}, {
+		"name": "Jormungand",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/11/73280.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "APP"
+			}]
+		}]
+	}, {
+		"name": "Kimi to Boku. 2",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/10/42107.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "merritt-10011"
+			}]
+		}]
+	}, {
+		"name": "Kore wa Zombie Desu ka? of the Dead OVA",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/4/43515.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "CrimsonV-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Mc-Fs"
+			}]
+		}]
+	}, {
+		"name": "Kuroko no Basket 2nd Season",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/9/56155.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "CheeZe-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "CrimsonV-Speed"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Darkwing-fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "pandoramon"
+			}]
+		}]
+	}, {
+		"name": "Medaka Box",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/13/37947.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "ACE-Z"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Muttsurini-FS"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "Neets-Fs"
+			}]
+		}, {
+			"status": "release",
+			"detail": [{
+				"name": "SBSubs"
+			}]
+		}]
+	}, {
+		"name": "Natsu-iro Kiseki",
+		"image": "https://myanimelist.cdn-dena.com/images/anime/5/53913.jpg",
+		"group": [{
+			"status": "release",
+			"detail": [{
+				"name": "DW"
+			}, {
+				"name": "Emergency-FS"
+			}]
+		}, {
+			"status": "release",
```

Comment:
from menome data Emergency-FS was alone release until EP.6 and since EP.7 is co-sub with DW
so it must be DW&Emergency-FS